app/client/src/features/dashboard/Dashboard.tsx

```diff
@@ -77,34 +44,11 @@ export function Dashboard({ queryRef }: Props) {
         if (!isLoading && !user) router.push('/');
```

Comment:
I suggest moving this logic to `app/client/src/pages/dashboard.tsx` for consistency, that way the auth status is being checked at the page level rather than the feature level since this is a protected route. If you look under `app/client/src/pages/organizations/me.tsx` for example you can see the check being performed there.